tests/csi/cstor/volume/provision_test.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	sc "github.com/openebs/maya/pkg/kubernetes/storageclass/v1alpha1"
+	spc "github.com/openebs/maya/pkg/storagepoolclaim/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("[cstor] TEST VOLUME PROVISIONING", func() {
+	var (
+		err     error
+		pvcName = "cstor-volume-claim"
+	)
+
+	BeforeEach(func() {
```

Comment:
Are we making use of a unique namespace?
The name should be `test-csi-cstor-prov-<uuid>`
Store this name against a global variable.
Make use of this throughout.

All the resources should make use of this namespace only.
Any cluster scoped resources should have same naming criteria i.e. `test-csi-cstor-prov-<uuid>`

Reply:
resolved